test/plugin/scenarios/guava-eventbus-scenario/support-version.list

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+19.0
+20.0
+21.0
+22.0
+23.0
+24.0-jre
+25.0-jre
+26.0-jre
+27.0-jre
+28.0-jre
+29.0-jre
+30.0-jre
+31.0-jre
```

Comment:
Please pick some versions once all tests passed. This seems to cause too many resources.

Reply:
OK, I will pick some versions. All versions tests have passed.